lib/Mojo/UserAgent.pm

```diff
@@ -385,7 +385,9 @@ Mojo::UserAgent - Non-blocking I/O HTTP and WebSocket user agent
   my $tx = $ua->put('[::1]:3000' => {'Content-Type' => 'text/plain'} => 'Hi!');
 
   # Quick JSON API request with Basic authentication
-  my $value = $ua->get('https://sri:t3st@example.com/test.json')->result->json;
+  use Mojo::URL;
```

Comment:
I don't like that this example contains a `use` line. That doesn't feel right for a synopsis.

Reply:
Would you prefer it just be omitted? (The example for the prepare event does this already)

Reply:
Removed the use line